packages/forms/src/UIForm/utils/validation.js

```diff
@@ -70,3 +173,68 @@ export function isValid(schema, errors) {
 
 	return true;
 }
+
+/**
+ * Filter the errors on array which items indexes are between a range
+ * This returns only the errors keys.
+ * @param errors The errors map
+ * @param arrayKey The array key
+ * @param minIndex The min item index (INCLUDED)
+ * @param maxIndex The max item index (EXCLUDED)
+ */
+export function filterArrayErrorsKeys(errors, arrayKey, minIndex, maxIndex) {
+	const minArrayIndexKey = Number.isInteger(minIndex) && arrayKey.concat(minIndex).toString();
+	const maxArrayIndexKey = Number.isInteger(maxIndex) && arrayKey.concat(maxIndex).toString();
+
+	return Object.keys(errors)
+		.filter(errorKey => errorKey.startsWith(arrayKey))
+		.filter(errorKey => !minArrayIndexKey || errorKey >= minArrayIndexKey)
+		.filter(errorKey => !maxArrayIndexKey || errorKey < maxArrayIndexKey);
```

Comment:
It seemed clearer to me like that, we have a filter for each single condition, easier to understand.
We could write 
```
return Object.keys(errors)
		.filter(isOnTargetArray)
		.filter(isAfterMinIndex)
		.filter(isBeforeMaxIndex);
```